pkgs/servers/roon-bridge/default.nix

```diff
@@ -1,4 +1,5 @@
-{ alsa-lib
+{ callPackage
```

Comment:
This makes no sense, you add `callPackage` as an input, but it doesn't get used anywhere.

Reply:
I was looking at a another package and misunderstood sorry about the callpackage